nodepy/twostep_runge_kutta_method.py

```diff
@@ -138,8 +122,8 @@ def stability_matrix(self,z):
         if self.type=='Type II':
             ahat = np.zeros([self.s,1]); ahat[:,0] = self.Ahat[:,0]
             bh = np.zeros([1,1]); bh[0,0]=self.bhat[0]
-            A = np.hstack([ahat,self.A])
-            A = np.vstack([np.zeros([1,self.s+1]),A])
+            A=np.hstack([ahat,self.A])
```

Comment:
Please don't remove the spaces around equal signs.  Actually, it would be better to do the opposite (add spaces where they are missing).  But I try to avoid mixing reformatting changes and feature changes in a single PR, since it makes it harder to pick out what is significant.